radicle-cli/src/terminal/io.rs

```diff
@@ -87,16 +87,16 @@ pub fn help(name: &str, version: &str, description: &str, usage: &str) {
 }
 
 pub fn usage(name: &str, usage: &str) {
-    eprintln!(
+    println!(
         "{} {}\n{}",
         style("==").red(),
         style(format!("Error: rad-{name}: invalid usage")).red(),
         style(prefixed(TAB, usage)).red().dim()
     );
 }
 
-pub fn eprintln(prefix: impl fmt::Display, msg: impl fmt::Display) {
-    eprintln!("{prefix} {msg}");
+pub fn println(prefix: impl fmt::Display, msg: impl fmt::Display) {
```

Comment:
These changes allow us to test the unhappy paths in the CLI examples, right?

I noticed I couldn't do that in the `rad id` tests.

Reply:
Yes exactly! It didn't really make sense for half the output to be in stdout and the other half in stderr.